File: android/app/src/main/java/com/lanbridge/app/core/enums/CapturerType.kt
package com.lanbridge.app.core.enums

/**
 * 采集源类型枚举
 */
enum class CapturerType {
    /** 麦克风采集 */
    Microphone,

    /** 系统音频采集（MediaProjection） */
    SystemAudio,

    /** 混音（麦克风 + 系统音频） */
    Mixed
}

File: windows/LanAudioBridge.Desktop/Core/Enums/CapturerType.cs
namespace LanAudioBridge.Core;

/// <summary>
/// 采集源类型枚举
/// </summary>
public enum CapturerType
{
    /// <summary>麦克风采集</summary>
    Microphone,

    /// <summary>系统音频采集（MediaProjection / WASAPI Loopback）</summary>
    SystemAudio,

    /// <summary>混音（麦克风 + 系统音频）</summary>
    Mixed
}
